tinyagentos/library_collections.py

```
@@ -0,0 +1,120 @@
+                text_content,
+                agent=f"library-{item_id[:12]}",
+                project=project_id,
+            )
+            handed_off += 1
+            logger.debug("Library item %s artifact %s ingested into taosmd",
+                         item_id, art["kind"])
+        except ImportError:
+            logger.debug("taosmd not available — collection indexing skipped")
+            # Still count as handed off since file is in place
+            handed_off += 1
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)